mmdet/apis/ote/__init__.py

```diff
@@ -12,4 +12,6 @@
 # See the License for the specific language governing permissions
 # and limitations under the License.
 
-from .extension import EpochRunnerWithCancel, EarlyStoppingHook, ReduceLROnPlateauLrUpdaterHook
+from .extension import EarlyStoppingHook
```

Comment:
why not 
```
from .extension import (
    EarlyStoppingHook,
    EpochRunnerWithCancel,
    ReduceLROnPlateauLrUpdaterHook,
)
```

Reply:
It's easier to track/change imports this way. But this is just for the transition period, good to return formatting back once we done.

Reply:
ok,
but if one there is one import per line inside brackets/tuple, and each import ended by comma (including the last), this also seems ok to track (to me)

Reply:
It is harder to change import for a single entity.

Reply:
agree